extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1209,6 +1219,39 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       this.win.removeEventListener('message', fluidMessageListener_);
     }
   }
+
+  /**
+   * Emits a postMessage containing information about this slot to the DFP
+   * Troubleshoot UI.
+   */
+  postTroubleshootMessage_() {
+    if (!this.win.opener || !/\?.*dfpdeb/.test(this.win.location.href)) {
```

Comment:
If you're looking for query string content, you can use this.win.location.search which includes the '?'... Doesn't actually save you much here but better to be explicit.  Also, do you want to handle edge cases like foodfpdeb or dfpdebfoo?  If so, should this be: /[\?|&]dfpdeb=/

Reply:
Fixed.